src/view/seed.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import MainContent from '../component/main-content';
+import { CopyText, H1Text, Text } from '../component/text';
+import { SplitBackground } from '../component/background';
+import { GlasButton } from '../component/button';
+import { color, font } from '../component/style';
+
+//
+// Seed View
+//
+
+const SeedView = ({ store }) => (
+  <SplitBackground image="purple-gradient-bg" bottom={color.blackDark}>
+    <CopySection />
+    <MainContent>
+      <WordList seedMnemonic={store.seedMnemonic.slice()} />
+      <GlasButton onPress={() => {}}>Next</GlasButton>
+    </MainContent>
+  </SplitBackground>
+);
+
+SeedView.propTypes = {
+  store: PropTypes.object.isRequired,
+};
+
+//
+// Copy Section
+//
+
+const copyStyles = StyleSheet.create({
+  wrapper: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  title: {
+    marginBottom: 15,
+  },
+  copyTxt: {
+    textAlign: 'center',
+    maxWidth: 410,
+  },
+});
+
+const CopySection = () => (
+  <View style={copyStyles.wrapper}>
+    <H1Text style={copyStyles.title}>First, write down your seed</H1Text>
+    <CopyText style={copyStyles.copyTxt}>
+      A seed is a group of words that will help you to recover your wallet if
+      you lose your password or computer. Write it down, and keep it in a safe
+      place.
+    </CopyText>
+  </View>
+);
+
+//
+// Word List
+//
+
+const listStyles = StyleSheet.create({
+  wrapper: {
+    flex: 1,
+    justifyContent: 'center',
+    margin: 20,
+  },
+  words: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    justifyContent: 'center',
+    maxWidth: 700,
+  },
+});
+
+const WordList = ({ seedMnemonic }) => (
+  <View style={listStyles.wrapper}>
+    <View style={listStyles.words}>
+      {seedMnemonic.map((word, i) => (
+        <Word word={word} index={i + 1} key={i} />
+      ))}
+    </View>
+  </View>
```

Comment:
This wasn't rendering correctly. So I changed the styled. Also `key` is a reserved attribute and can't be used as a prop...

![image](https://user-images.githubusercontent.com/1374174/41222989-d2c8c408-6d68-11e8-9584-fbd35d2db669.png)

Reply:
Hmm, that's weird, i was getting a console error saying `key` was required when you're using a `for` loop like that... i'll re-run it